src/internal_modules/roc_core/parse_duration.h

```diff
@@ -33,6 +33,19 @@ namespace core {
 //!  false if string can't be parsed.
 ROC_ATTR_NODISCARD bool parse_duration(const char* string, nanoseconds_t& result);
 
+//! Parse size from string.
+//!
+//! @remarks
+//!  The input string should be in one of the following forms:
+//!   - "<number>"
+//!   - "<number>K"
+//!   - "<number>M"
+//!   - "<number>G"
```

Comment:
I've supported G, not sure if it's needed - can easily remove.

Reply:
Let's keep it :)